### DISCUSSION TITLE
Unwanted trailing / when Console emits webhook

### DISCUSSION THREAD

**Discussion by ghost** [mia-platform/community#398] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v12.2.0

### Description

I've configured a new Webhook into my Company to be emitted by the Console each time a create a new service. The configurated endpoint is something like:
`https://host/uri`

The problem is that the API Gateway that I use to expose this endpoint is receiving this request instead:
`https://host/uri/`
so with an unwanted trailing / at the end of the endpoint.

Is this a bug or a wanted behaviour?
Thank you!

### Actual Outcome

Configured webhook to be emitted when new services gets created, exposed endpoint to manage the incoming request, the Console is perhaps calling the endpoint with a trailing `/` so the API GW returns a 404

### Expected Outcome

The Console calls the configured endpoint in the Webhook section, without the trailing `/`

**Comment by fredmaggiowski** [comment]
Sounds like a bug to me but let's hear from @Danielecina

**Comment by mattia-fumo** [comment, marked as answer]
This is a known bug which has been fixed in the next Console version (>=v12.2.1)
